docs/pages/database-access/guides/redshift-serverless.mdx

```diff
@@ -0,0 +1,400 @@
+---
+title: Database Access with Redshift Serverless on AWS
+description: How to configure Teleport Database Access with AWS Redshift Serverless.
+---
+
+This guide will help you to:
+
+- Set up Teleport to access your AWS Redshift Serverless instances.
+- Connect to your databases through Teleport.
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+![Teleport Database Access Redshift Self-Hosted](../../../img/database-access/guides/redshift_selfhosted_serverless.png)
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+![Teleport Database Access Redshift Cloud](../../../img/database-access/guides/redshift_cloud_serverless.png)
+</ScopedBlock>
```

Comment:
I think we should also include the STS service in these images (see the old dynamodb guide for example).

Reply:
I can do that. For my own edification, what is the STS service, and how does it relate to this guide?

Reply:
sts is the Security Token Service that we make the AssumeRole call to impersonate the access role for the teleport user.

Reply:
Updated the images in a3d1684df72a0b5a16b3d08ef993f286847c1c37